tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
```

Comment:
We want to consistently use `es5` as our target on FE libraries for largest browser and external dependency support.